keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -234,54 +234,77 @@ implementation.  The "Design Details" section below is for the real
 nitty-gritty.
 -->
 #### Terminology
+- **multicluster namespace** - A placeholder name for a group of
```

Comment:
I'm very name-coining averse in the area of multicluster so I have some hesitation about this name. Basically no matter what name we choose for anything in this area, there is a chance that someone will reason incorrectly about it given implications of the name. How about the term 'client namespace' instead here? I would hate for someone to read into this an implication that there will be some technology called 'multicluster namespaces' that comes out of this SIG. I know there is a note below, but snippets of this doc might be pasted that don't include the note.

I hope that doesn't come across as nitpicky, I have had many many years of dispelling mistaken impressions about the multicluster area of kube community that were based on such assumptions folks had (understandably) made.